logins-api/android/library/src/main/java/mozilla/appservices/logins/rust/PasswordSyncAdapter.kt

```diff
@@ -11,16 +11,25 @@ import com.sun.jna.Library
 import com.sun.jna.Native
 import com.sun.jna.Pointer
 import com.sun.jna.PointerType
+import java.lang.reflect.Proxy
 
 
 @Suppress("FunctionNaming", "TooManyFunctions", "TooGenericExceptionThrown")
 internal interface PasswordSyncAdapter : Library {
     companion object {
+        // If our shared object isn't available, error when you attempt to call our methods
+        // intead of at startup.
         private const val JNA_LIBRARY_NAME = "loginsapi_ffi"
-        internal var INSTANCE: PasswordSyncAdapter
 
-        init {
-            INSTANCE = Native.loadLibrary(JNA_LIBRARY_NAME, PasswordSyncAdapter::class.java) as PasswordSyncAdapter
+        internal var INSTANCE: PasswordSyncAdapter = try {
```

Comment:
Hmm, interesting.  I'm fine with this, since it's clearly helping folks in the wild; but my general preference is, of course, to do it right: #262.  But in the interim, rock on!